plugin.video.ardaktuell/resources/lib/ardaktuell/ardaktuelladdon.py

```diff
@@ -0,0 +1,138 @@
+
+
+class ArdAktuellAddon(AbstractRssAddon):
+
+    __PLUGIN_ID__ = "plugin.video.ardaktuell"
+
+    QUALITY_LEVEL = ["webxl", "webl", "webm", "webs"]
+
+    BROADCASTS = [
+        {
+            "name": "ARD Tagesschau",
+            "icon": "https://www.tagesschau.de/image/podcast/ts-1400.jpg",
+            "video_url": "https://www.tagesschau.de/export/video-podcast/%s/tagesschau_https/",
+            "audio_url": "https://www.tagesschau.de/export/podcast/tagesschau_https/"
+        },
+        {
+            "name": "ARD Tagesschau mit Gebärdensprache",
+            "icon": "https://www.tagesschau.de/image/podcast/tsg-1400.jpg",
+            "video_url": "https://www.tagesschau.de/export/video-podcast/%s/tagesschau-mit-gebaerdensprache_https/"
+        },
+        {
+            "name": "ARD Tagesschau in 100 Sekunden",
+            "icon": "https://www.tagesschau.de/image/podcast/ts100s-1400.jpg",
+            "video_url": "https://www.tagesschau.de/export/video-podcast/%s/tagesschau-in-100-sekunden_https/",
+            "audio_url": "https://www.tagesschau.de/export/podcast/hi/tagesschau-in-100-sekunden/"
+        },
+        {
+            "name": "ARD Tagesthemen",
+            "icon": "https://www.tagesschau.de/image/podcast/tt-1400.jpg",
+            "video_url": "https://www.tagesschau.de/export/video-podcast/%s/tagesthemen_https/",
+            "audio_url": "https://www.tagesschau.de/export/podcast/tagesthemen_https/"
+        },
+        {
+            "name": "ARD Nachtmagazin",
+            "icon": "https://www.tagesschau.de/image/podcast/nm-1400.jpg",
+            "video_url": "https://www.tagesschau.de/export/video-podcast/%s/nachtmagazin_https/",
+            "audio_url": "https://www.tagesschau.de/export/podcast/nachtmagazin_https/"
+        },
+        {
+            "name": "ARD Bericht aus Berlin",
+            "icon": "https://www.tagesschau.de/image/podcast/bab-1400.jpg",
+            "video_url": "https://www.tagesschau.de/export/video-podcast/%s/bab_https/",
+            "audio_url": "https://www.tagesschau.de/export/podcast/bab_https/",
+            "type": "video"
+
+        },
+        {
+            "name": "ARD Tagesschau vor 20 Jahren",
+            "icon": "https://www.tagesschau.de/image/podcast/tsv20-1997-1400.jpg",
+            "video_url": "https://www.tagesschau.de/export/video-podcast/%s/tagesschau-vor-20-jahren_https/",
+            "audio_url": "https://www.tagesschau.de/export/podcast/tagesschau-vor-20-jahren_https/",
+            "type": "video"
+        },
+        {
+            "name": "ARD Mal angenommen...",
+            "icon": "https://www.tagesschau.de/multimedia/bilder/mal-angenommen-podcast-cover-105~_v-original.jpg",
+            "audio_url": "https://www.tagesschau.de/multimedia/podcasts/mal-angenommen-feed-101.xml",
+            "type": "audio"
+        }
+    ]
+
+    def __init__(self, addon_handle):
+
+        AbstractRssAddon.__init__(self, self.__PLUGIN_ID__, addon_handle)
```

Comment:
`super().__init__(self.__PLUGIN_ID__, addon_handle)`. It's Python 3 after all :-)

Reply:
I'm going to change it next time.